<launch>
  <group ns="usb_cam">
	<node pkg="nodelet" type="nodelet" name="camera_nodelet_manager" args="manager" />

	<node pkg="nodelet" type="nodelet" name="camera_nodelet"
          args="load usb_cam/UsbCamNodelet camera_nodelet_manager" >
      <param name="video_device" value="/dev/video0" />
      <param name="image_width" value="800" />
      <param name="image_height" value="600" />
      <param name="pixel_format" value="yuyv" />
      <param name="camera_frame_id" value="usb_cam" />
      <param name="io_method" value="mmap"/>
      <!--param name="camera_info_url" if="$(arg calibrated)"
             value="file://$(env HOME)/.ros/camera_info/$(arg camera_serial).yaml" /-->
    </node>
  </group>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>
</launch>
